bellows/zigbee/zcl/foundation.py

```diff
@@ -101,7 +101,6 @@ class Discrete:
 DATA_TYPE_IDX = {
     t: tidx
     for tidx, (tname, t, ad) in DATA_TYPES.items()
-    if ad is Analog
```

Comment:
I was trying to write a boolean on an On/Off cluster.

Should these discrete values be handled in a other way?